account_bank_statement_import_move_line/README.rst

```diff
@@ -34,10 +34,19 @@ statement.
 .. contents::
    :local:
 
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Settings > Users & Companies > Users*.
+#. Verify or add group *Show Full Accounting Features*
+   on the user or users that need use this feature.
+
 Usage
 =====
 
-#. Go to Invoicing > Dashboard.
+#. Go to Invoicing > Configuration > Accounting > Journals.
```

Comment:
Why changing this path? In v12 is the same.

Reply:
Perhaps I am wrong but following previous steps didn't get me to the desired object. I would try again and provide an animated gif if still see that way

Reply:
![Invoicing_navigation](https://user-images.githubusercontent.com/16461806/63420551-2d940000-c407-11e9-8b49-78d8679c3bd3.gif)

@pedrobaeza Sorry but I am not sure how could I get through to Journals from the Invoicing > Dashboard menu

Reply:
That's because in your user, you don't have "Show full accounting features" option.